pypeit/images/pypeitimage.py

```diff
@@ -52,6 +53,7 @@ class PypeItImage(datamodel.DataContainer):
             Image count scaling applied (e.g., 1/flat-field)
         bpm (`numpy.ndarray`_, optional):
             Bad pixel mask
+        # TODO This docs need to indicate the types of things. Are crmask fullmask etc. boolean or ints?
```

Comment:
Can you move this outside of the doc string so that it doesn't appear on readthedocs?